### PR TITLE
feat(content): Flesh out "Fixing Common Errors" with more examples

### PR DESCRIPTION
**Pull request by brody192**
adds some keyword to help when searching

### CODE REVIEW

File: src/docs/troubleshoot/fixing-common-errors.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ users commonly encounter.
 
 ## Application Error: This application failed to respond
 
+https requets will fail with status code 503 Bad Gateway
```
Comment (rob93c) [inline src/docs/troubleshoot/fixing-common-errors.md:30]: There's a typo
```suggestion
The HTTPS request will fail with status code 503 Bad Gateway
```

File: src/docs/troubleshoot/fixing-common-errors.md [hunk 2]
```diff
@@ -61,21 +63,80 @@ at the port you specified. For more information, check out
...
+// Listen on `port` and 0.0.0.0
+async function bootstrap() {
+  // ...
+  await app.listen(port, "0.0.0.0");
+}
+```
+
+### Node / Next
+
+next needs an edditional flag to listen on `port`
```
Comment (rob93c) [inline src/docs/troubleshoot/fixing-common-errors.md:95]:
```suggestion
next needs an additional flag to listen on `port`
```

File: src/docs/troubleshoot/fixing-common-errors.md [hunk 3]
```diff
@@ -61,21 +63,80 @@ at the port you specified. For more information, check out
 **This approach is not recommended**.
 </Banner>
 
-#### Node/Express
+## Some examples for a few frameworks / servers:
```
Comment (rob93c) [inline src/docs/troubleshoot/fixing-common-errors.md:66]: The elements of this section should all share the same heading (`###`) to have a consistent result
So, the following will need to be amended accordingly:
* Node / Express
* Node / Nest
* Python / Gunicorn

**Review by rob93c — CHANGES_REQUESTED** [review]
Hi @brody192, I just went though your changes and I found a few minor things to sort out

File: src/docs/troubleshoot/fixing-common-errors.md (hunk 3 above)
Comment (brody192) [reply src/docs/troubleshoot/fixing-common-errors.md:66]: this was intentional

**Review by rob93c — APPROVED** [review]

File: src/docs/troubleshoot/fixing-common-errors.md [hunk 4]
```diff
@@ -61,21 +63,80 @@ at the port you specified. For more information, check out
...
+func main() {
+  // ...
+
+  // Use PORT provided in environment or default to 3000
+  var port = envPortOr("3000")
+
+  log.Fatal(http.ListenAndServe(port, handler))
+}
+
+// returns environment PORT or port parameter, both will be prfixed with ":"
```
Comment (half0wl) [inline src/docs/troubleshoot/fixing-common-errors.md:132]:
```suggestion
// returns environment PORT or port parameter, both will be prefixed with ":"
```

File: src/docs/troubleshoot/fixing-common-errors.md (hunk 3 above)
Comment (half0wl) [reply src/docs/troubleshoot/fixing-common-errors.md:66]: Also curious about this, are you going for "most common language/framework" as the header?

I think having them consistent with h3/`###` makes sense

File: src/docs/troubleshoot/fixing-common-errors.md [hunk 5]
```diff
@@ -27,6 +27,8 @@ users commonly encounter.
 
 ## Application Error: This application failed to respond
 
+The HTTPS request will fail with status code 503 Bad Gateway
```
Comment (half0wl) [inline src/docs/troubleshoot/fixing-common-errors.md:30]: Super nitpick: I'd add this below the screenshot so that it reads better

**Review by half0wl — CHANGES_REQUESTED** [review]
@brody192 thanks! Sorry it took me so long to get to this.

**Pull request merged**